### PR TITLE
chore: Revisit and fix failing examples

### LINKED ISSUE #949: Revisit and fix failing examples
### Problem

Currently some examples are not working as expected.

### Solution

Revisit and fix failing examples. Also add a check in the CI that runs the examples if possible.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by 0xivanov**
**Description**:
Revisit and fix failing examples.

This PR fixes:
1. PrngTransaction
2. Client's default localhost mirror node address
3. Adds topic subscribe tests to the CI
4. Adds the examples to the CI

**Related issue(s)**:

Fixes #949

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by jeromy-cannon — APPROVED** [review]

File: examples/zero_token_operations/main.go [hunk 1]
```diff
@@ -114,10 +123,11 @@ func main() {
 		panic(err)
 	}
 
-	_, err = helper.ExecuteSteps(0, 5, client)
-	if err != nil {
-		panic(fmt.Sprintf("%v : error in helper", err))
-	}
+	// TODO there is currently possible bug in services causing this operation to fail, should be investigated
```
Comment (gsstoykov) [inline examples/zero_token_operations/main.go:126]: Maybe we need to add a test for this and keep it monitored.

**Review by gsstoykov — DISMISSED** [review]

File: examples/zero_token_operations/main.go (hunk 1 above)
Comment (0xivanov) [reply examples/zero_token_operations/main.go:126]: I will create issue about this once i've investigated it.

File: topic_message_query_e2e_test.go [hunk 2]
```diff
@@ -83,75 +83,78 @@ In consequat, nisi iaculis laoreet elementum, massa mauris varius nisi, et porta
...
+
+	_, err = NewTopicMessageQuery().
+		SetTopicID(topicID).
+		SetStartTime(time.Unix(0, 0)).
+		SetLimit(1).
+		SetCompletionHandler(func() {
+			atomic.StoreInt32(&finished, 1)
+		}).
+		Subscribe(env.Client, func(message TopicMessage) {
+			// Do nothing
```
[the remark this answers is not in the document]
Comment (0xivanov) [reply topic_message_query_e2e_test.go]: removed

File: Taskfile.yml [hunk 3]
```diff
@@ -38,14 +38,19 @@ tasks:
...
-                            echo "Running $example/main.go"
-                            env OPERATOR_KEY="302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137" OPERATOR_ID="0.0.2" HEDERA_NETWORK="localhost" go run main.go
-                            fi
-                        fi
-                        popd > /dev/null
-                    fi
-                done
+              for example in examples/*; do
+                  if [ -d "$example" ]; then
+                      # Skip the consensus_pub_sub_chunked directory
```
Comment (0xivanov) [inline Taskfile.yml:43]: This does not skip the example

**Review by gsstoykov — APPROVED** [review]

**Review by gsstoykov — APPROVED** [review]
LGTM.

**Pull request merged**